docs/manual/java/guide/broker/KafkaClient.md

```diff
@@ -4,7 +4,23 @@ Lagom provides an implementation of the Message Broker API that uses Kafka. In t
 
 ## Dependency
 
-To use this feature add the following in your project's build.
+You may need to use Kafka to publish data and/or you may need to subscribe to a topic providing data from third parties. When your service acts only as a Topic Subscriber you can import the Lagom Kafka Client alone.
```

Comment:
There are really three (or even four) cases to cover

1. A service that both produces a topic and consumes another topic
2. A service that only produces a topic (which isn't really different than 1 as far as dependencies go)
3. A service that only consumes a topic produced by another Lagom service
4. A service that consumes a topic produced by a non-Lagom service

I'd suggest removing the reference to "third parties" as that may be misleading, and restructuring this so that the original broker dependency is listed first as the default case, with this section moved after it with an introduction like "Services that only need to subscribe to one or more topics published by other services or non-Lagom producers, you can import the Lagom Kafka Client alone."

It would also be good to document how to subscribe to non-Lagom topics (by creating a stub service interface), since it has come up a couple of times recently.